README.md

```diff
@@ -574,9 +574,8 @@ See the recommended enabled configuration below:
       "max-split-routes": 3,
       // Maximum number of iterations to split a route across.
       "max-split-iterations": 10,
-      // Minimum liquidity for a pool to be included in the router
-      // denominated in OSMO.
-      "min-osmo-liquidity": 100,
+      // Minimum liquidity capitalization for a pool to be considered in the router.
```

Comment:
```suggestion
      // Minimum liquidity capitalization in USDC for a pool to be considered in the router.
```

Reply:
I would like to keep this more general because it is to be eventually set by the following config:
https://github.com/osmosis-labs/sqs/blob/187ad1b75d6605b7ec80bec4601e77b49c8d7648/config.json#L37

So it might not always be USDC, at some point we might switch to USDT.

I've aimed to explain this in a few places but let me also add some comments here:
[0f3ccd3](https://github.com/osmosis-labs/sqs/pull/241/commits/0f3ccd3fcb7c69949c599bc0fc4df2b0fc3b3e4e)

Reply:
Small suggestion - I'd like to know the unit when making changes to a config

Reply:
Does the comment [I added](https://github.com/osmosis-labs/sqs/blob/f6a175fa550d2aa765b3be01280ed0639fd54a27/README.md?plain=1#L578) about the [pricing.default-quote-human-denom](https://github.com/osmosis-labs/sqs/blob/f6a175fa550d2aa765b3be01280ed0639fd54a27/README.md?plain=1#L604) address your request? 


From that comment, it should be clear it is USDC.

I'll apply the recommendation if you think the change I made is insufficient